BrainPortal/lib/boutiques_file_name_verifier.rb

```diff
@@ -0,0 +1,67 @@
+
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2024
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# This module adds automatic verification of the
+# files (or directories ) names generated as output, which comply with CBRAIN restrictions.
+#
+# For example, in the "inputs" section:
+#
+#   {
+#     "description": "The name of the folder to store outputs of XCPD processing.",
+#     "id": "output_dir",
+#     "name": "output_dir",
+#     "optional": false,
+#     "type": "String",
+#     "value-key": "[OUTPUT_DIR]",
+#     "default-value": "xcpd_output"
+#   }
+#
+# "cbrain:integrator_modules": {
```

Comment:
Indent the JSON a bit further, int he comment block it now starts at the same level as the text, so it would not be rendered as a code block in the auto doc.

Reply:
line 38 was deleted as per a previous comment

Reply:
I know. I mean the JSON is not indented properly anymore.